movies/views.py

```diff
@@ -42,4 +43,58 @@ def get(self,request):
             "poster_image"   : movie.poster_image
         } for movie in movies ][OFFSET : LIMIT]
 
-        return JsonResponse({"MOVIE_LIST" : movie_list}, status=200)
\ No newline at end of file
+        return JsonResponse({"MOVIE_LIST" : movie_list}, status=200)
+
+
+class CommentView(View):
+    def post(self, request, movie_id):
+        try:    
+            data = json.loads(request.body)
+            user = User.objects.filter(id=data["user_id"])
+            movie = Movie.objects.filter(id=movie_id)
+
+            if not movie.exists():
+                return JsonResponse({"MESSAGE" : "MOVIE_DOES_NOT_EXIST"}, status=404)
+            
+            if not user.exists():
+                return JsonResponse({"MESSAGE" : "USER_DOES_NOT_EXIST"}, status=404)
+            
+            if not Rating.objects.filter(user_id=data["user_id"]).exists():
+                return JsonResponse({"MESSAGE" : "ENTER_RATING_FIRST"}, status=404)
+
+            Rating.objects.create(
+                user_id    = User.objects.get(id=data["user_id"]).id,
+                movie_id   = Movie.objects.get(id=movie_id).id,
```

Comment:
이미 위에서 movie_id 가 존재함을 확인했기 때문에, get 으로 사 영화 정보를 호출하지 않아도 괜찮아보입니다.
```suggestion
                movie_id   = movie_id
```

Reply:
수정사항 반영하였습니다.